crates/status/src/impl/token_status_list/json.rs

```diff
@@ -238,6 +239,10 @@ impl StatusMapEntry for StatusListReference {
     fn status_list_url(&self) -> &iref::Uri {
         &self.uri
     }
+
+    fn status_size(&self) -> Option<Self::StatusSize> {
+        None
```

Comment:
Right, we could (and we should I'll change that). It's just not needed because the status size is also stored in the status list.

Reply:
yeah I get that, put differently I guess to me it feels like the common interface doesn't really make sense anymore when they diverge in this way. I would be pretty confused if there was a status_size for the common interface that returned None because it's a bitstring, since it conceptually _does_ have a size, it's just stored in a different place in the underlying data structure.

Reply:
In other words it's not clear to me why it would be useful to use the common interface at all if it doesn't abstract away details like _where_ the status size is found? Maybe an option would be to remove the common interface (or make it non-public & comment that it's in development)?

Reply:
In practice it does because this method is not intended to be used directly by the user. To get a status from an entry, users are supposed to use the `StatusMap::get_entry` method that uses the `StatusMapEntry::status_size` method to see if the entry knows the status size. If it does not, we assume the status list knows the status size.

But I agree, because of this difference between the two specs the common interface is unintuitive, and the lack of documentation doesn't help :cry: